src/server.js

```diff
@@ -0,0 +1,10 @@
+const logger = require('~logger');
+const server = require('./index');
+
+const { API_PORT, API_URL } = process.env;
```

Comment:
👍 +1 for logic extraction. But please remove unused var `API_URL`.

Reply:
Thanks for noticing. Is it okay if I add a pre-commit hook that will run ESLint to avoid these kinds of issue?

Reply:
Sure. That'll be great. 👍